lib/erb_lint/cli.rb

```diff
@@ -157,9 +158,19 @@ def load_options(args)
     def lint_files
       if @options[:lint_all]
         pattern = File.expand_path(DEFAULT_LINT_ALL_GLOB, Dir.pwd)
-        Dir[pattern]
+        Dir[pattern].select { |filename| !excluded?(filename) }
       else
-        @files.map { |f| f.include?('*') ? Dir[f] : f }.flatten.map { |f| File.expand_path(f, Dir.pwd) }
+        @files
+          .map { |f| f.include?('*') ? Dir[f] : f }
+          .flatten
+          .map { |f| File.expand_path(f, Dir.pwd) }
+          .select { |filename| !excluded?(filename) }
+      end
+    end
+
+    def excluded?(filename)
```

Comment:
What do you think about moving this method to the linter class and let each linters decided whether or not to run ?

Reply:
There is already a similar method in `LinterConfig` that checks each linter's specific exclusion list, but this one only checks the global exclusions. This is mainly to get an accurate list of files to lint, it will save time reading and parsing files excluded by each linter.